packages/relay/src/lib/constants.ts

```diff
@@ -145,7 +145,7 @@ export default {
   // @ts-ignore
   HBAR_RATE_LIMIT_DURATION: parseInt(ConfigService.get('HBAR_RATE_LIMIT_DURATION') || '86400000'), // 1 day
   // @ts-ignore
-  HBAR_RATE_LIMIT_TOTAL: BigNumber(ConfigService.get('HBAR_RATE_LIMIT_TINYBAR') || '800000000000'), // 8000 HBARs
+  HBAR_RATE_LIMIT_TOTAL: BigNumber(ConfigService.get('HBAR_RATE_LIMIT_TINYBAR') ?? '800000000000'), // 8000 HBARs
```

Comment:
Q: why this change and does it affect all the other configs?

Reply:
While working on the acceptance test I noticed that the zero value for the `HBAR_RATE_LIMIT_TINYBAR` was not getting picked up.  Most likely because the logical OR operator `||` returns the first truthy value and 0 is falsy.  The nullish coalescing operator `??` falls back to the default value when the left-hand operand is `null` or `undefined`, not when it's `0` or any other falsy value.

Reply:
Nice catch.
Can you make a note here on that. Also i think you should open a quick issue to note that all integer numbers will hit this issue so we should do a quick check and see if there are other items that need to be fixed in a separate PR